canvas_modules/common-canvas/src/common-properties/controls/textfield/textfield.jsx

```diff
@@ -51,6 +53,9 @@ class TextfieldControl extends React.Component {
 		if (this.isList) {
 			value = ControlUtils.splitNewlines(value, arrayValueDelimiter);
 		}
+		if (value.length === 0 && typeof this.defaultValue === "undefined") {
```

Comment:
hmm, we might also want to handle the null case as well.  a default value of null.

Reply:
updated!